### PR TITLE
Typos, some doc additions

### PR DESCRIPTION
**Pull request by mvhconsult**
Just clarification. I guess there is a need to check against current QGis version. The copyright suggests this is alll from 2012??

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -52,17 +54,19 @@ All major browsers should be supported.
...
+- Apache2 - Webserver (Ubuntu: apache2) with php enabled
 - mod-fcgid (Ubuntu: libapache2-mod-fcgid)
 - QGIS and QGIS Server (best installed from source)
 
 On ubuntu you can meet these requirements by simply doing:
 
 ``sudo apt-get install libapache2-mod-fcgid``
 
-The QGIS Server compilation and installation will be covered in the QGIS manual.
+The QGIS Server compilation and installation will is covered in the online QGIS manual. See http://www.qgis.org/en/docs/index.html for the current version.
```
Comment (DelazJ) [inline README.rst]: ~~will~~ is

**Comment by DelazJ** [comment]
General comment on the changes: do not merge lines over 80  characters; they are intentionally split around this value because short lines can be put side by side and  are easy to review and comment

**Comment by DelazJ** [comment]
General comment on the repo: is it a maintained repo (who is in charge)? Have never noticed a PR until this one. And how does this relate to the QWC v2?

**Comment by manisandro** [comment]
This is QWC v1, the predecessor.

**Comment by tudorbarascu** [comment]
I don't think it will be maintained any longer as the effort goes into qwc2

**Comment by mvhconsult** [comment]
If this means that is stale documentation, then my suggestion is to take it off line.

**Comment by andreasneumann** [comment]
We can add a note that the project is succeeded by QWC2 and not actively maintained.

I don't think it is a good move to take it offline, as there are still people using the old version. We can rethink this decision in 1-2 years.

**Pull request merged**